index.html

```diff
@@ -662,7 +662,45 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 the cost of implementation complexity.
       </dd>
     </dl>
-
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats can only reveal all
+attribute values in a verifiable credential.
+    </p>
+    <p>
+Zero-Knowledge Proofs (ZKPs) are a proof format that enables
+privacy-preserving and data-minimization features in
+verifiable presentations, such as selective disclosure and predicate proofs.
+    </p>
+    <p>
+Selective disclosure is the ability of a holder to reveal a subset of
+the attributes of a verifiable credential in a verifiable presentation.
+A presentation based on ZKPs only contains those attributes and
+associated values that are required to satisfy the presentation requirements. 
+    </p>
+    <p>
+ZKP predicate proofs are a type of proof associated with an attribute.
+Predicate proofs include comparisons, such as
+"greater than", "less than", "not equal", "range", 
+"set membership", and "set non-membership".
+Predicate proofs can be
+constructed without requiring the issuer to create special fields, such as
+"age-under-18", age-over-21", "age-over-60", in the verifiable credential
```

Comment:
Missing double-quote before `age-over-21",`

Reply:
Yes. I will update the text.

Reply:
Added @TallTed suggestion.